src/tito/tagger/main.py

```diff
@@ -393,48 +395,30 @@ def _bump_version(self, release=False, zstream=False):
             out_f = open(self.spec_file + ".new", 'w')
 
             for line in in_f.readlines():
-                if release:
-                    match = re.match(release_regex, line)
-                    if match:
-                        line = "".join((match.group(1),
-                                        increase_version(match.group(2)),
-                                        "\n"
-                        ))
-                elif zstream:
```

Comment:
This zstream path is the only one that worries me, looks like we may not have test coverage for that via the actual tagger tests. However the logic looks ok.